client-app/src/apps/contact.ts

```diff
@@ -2,8 +2,8 @@ import '../Bootstrap';
 
 import {XH} from '@xh/hoist/core';
 import {AppContainer} from '@xh/hoist/desktop/appcontainer';
-import {AppComponent} from '../examples/contact/AppComponent';
-import {AppModel} from '../examples/contact/AppModel';
+import {AppComponent} from '../examples/contact/desktop/AppComponent';
```

Comment:
You will se this update across this PR. 
After some back and forth, Lee suggested that I nest the new `mobile` contacts under the parent /examples/contacts, and do the same with `desktop`. 
This means that some of this PR will be just file path adjustments to accomodate this change.